frontend/packages/console-shared/src/components/popover/SizeAwarePopover.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Measure from 'react-measure';
+import { Popover, Button } from '@patternfly/react-core';
+import './SizeAwarePopover.scss';
+
+type SizeAwarePopoverProps = {
+  content: string;
+};
+
+const SizeAwarePopover: React.FC<SizeAwarePopoverProps> = ({ content }) => {
```

Comment:
First and foremost this component renders text and not a popover. Rename to `ClampedText`. Unless someone else has a better name :)